### PR TITLE
Add XSUM Datasheet

### PR DESCRIPTION
**Pull request by Shashi456**

### CODE REVIEW

**Comment by sebastianGehrmann** [comment]
Hey, are you planning to fill in the remaining sections in this PR or should I merge and you create a follow-up?

**Comment by Shashi456** [comment]
> Hey, are you planning to fill in the remaining sections in this PR or should I merge and you create a follow-up?

I was waiting for @shashiongithub's  review, but I think merging this right now would be fine.

File: web/tasks/XSum.md [hunk 1]
```diff
@@ -10,36 +10,43 @@ motivation: TODO.
...
 ## Dataset Description
 
-- **Homepage:**
-- **Repository:**
-- **Paper:**
-- **Point of Contact:**
+- **Homepage:** 
+- **Repository:** https://github.com/EdinburghNLP/XSum
+- **Paper:** [Original Paper](https://arxiv.org/abs/1808.08745)
+- **Point of Contact:** [Shashi Narayan](shashi.narayan@gmail.com)
```
Comment (shashiongithub) [inline web/tasks/XSum.md:16]: shashinarayan@google.com

File: web/tasks/XSum.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'XSum'
 type: Summarization
-motivation: TODO.
+motivation: [TODO]
```
Comment (shashiongithub) [inline web/tasks/XSum.md:4]: Large scale monolingual dataset for evaluating extreme summarization.

File: web/tasks/XSum.md [hunk 3]
```diff
@@ -10,36 +10,43 @@ motivation: TODO.
 
 ## Dataset Description
 
-- **Homepage:**
-- **Repository:**
-- **Paper:**
-- **Point of Contact:**
+- **Homepage:**
```
Comment (shashiongithub) [inline web/tasks/XSum.md:13]: NA (See Repository)

File: web/tasks/XSum.md [hunk 4]
```diff
@@ -10,36 +10,43 @@ motivation: TODO.
 
 ## Dataset Description
 
-- **Homepage:**
-- **Repository:**
-- **Paper:**
-- **Point of Contact:**
+- **Homepage:** 
+- **Repository:** https://github.com/EdinburghNLP/XSum
+- **Paper:** [Original Paper](https://arxiv.org/abs/1808.08745)
```
Comment (shashiongithub) [inline web/tasks/XSum.md:15]: https://www.aclweb.org/anthology/D18-1206

File: web/tasks/XSum.md [hunk 5]
```diff
@@ -10,36 +10,43 @@ motivation: TODO.
...
+- **Point of Contact:** [Shashi Narayan](shashi.narayan@gmail.com)
 
 ### Dataset and Task Summary
 
-[More Information Needed]
+The dataset is for the task of abstractive summarization in its extreme form, its about summarizing a document in a single sentence. It introduces extreme summarization, a new single-document summarization task which does not favor extractive strategies and calls for an abstractive modeling approach. The idea is to create a short, one-sentence news summary answering the question "What is the article about?".
 
 ### Why is this dataset part of GEM?
 
 [More Information Needed]
```
Comment (shashiongithub) [inline web/tasks/XSum.md:24]: This dataset is part of the GEM benchmark for the task of summarization, alongside MLSum and WikiLingua, and acts as a large-scale, high-quality resource for extreme summarization.

File: web/tasks/XSum.md [hunk 6]
```diff
@@ -10,36 +10,43 @@ motivation: TODO.
...
+```@InProceedings{xsum-emnlp,
+  author =      "Shashi Narayan and Shay B. Cohen and Mirella Lapata",
+  title =       "Don't Give Me the Details, Just the Summary! {T}opic-Aware Convolutional Neural Networks for Extreme Summarization",
+  booktitle =   "Proceedings of the 2018 Conference on Empirical Methods in Natural Language Processing ",
+  year =        "2018",
+  address =     "Brussels, Belgium",
+}
+```
 
 ### Leaderboard
```
Comment (shashiongithub) [inline web/tasks/XSum.md:52]: This dataset has no corresponding public leaderboard.

File: web/tasks/XSum.md [hunk 7]
```diff
@@ -72,11 +95,11 @@ motivation: TODO.
 
 #### Initial Data Collection and Normalization
 
-[More Information Needed]
+The dataset consists of BBC articles and accompanying single sentence summaries. Specifically, each article is prefaced with an introductory sentence (aka summary) which is professionally written, typically by the author of the article. They collected 226,711 Wayback archived BBC articles ranging over almost a decade (2010 to 2017) and covering a wide variety of domains (e.g., News, Politics, Sports, Weather, Business, Technology, Science, Health, Family, Education, Entertainment and Arts). Each article comes with a unique identifier in its URL, which was then used to randomly split the dataset into training (90%, 204,045), validation (5%, 11,332), and test (5%, 11,334) set. 
 
 #### Who are the source language producers?
 
-[More Information Needed]
+N/A (dataset authors)
```
Comment (shashiongithub) [inline web/tasks/XSum.md:102]: Professional journalists.

**Review by shashiongithub — COMMENTED** [review]
Could not leave comments on the bottom part. It might be good idea to provide details under 

## Getting started with in-depth research on the task

For example: see https://github.com/GEM-benchmark/GEM-benchmark.github.io/blob/main/web/tasks/WikiLingua.md

**Comment by sebastianGehrmann** [comment]
ping about this

**Comment by sebastianGehrmann** [comment]
ping again @Shashi456

**Comment by Shashi456** [comment]
I'll get in done asap, sorry for the delay.

**Comment by Shashi456** [comment]
@sebastianGehrmann Done, I'll add the baseline and links for the notebook after talking to the person who set up the baseline.

**Comment by sebastianGehrmann** [comment]
Okay, thank you! Merging for now

**Pull request merged**